phenex/filters/relative_time_range_filter.py

```diff
@@ -56,24 +56,7 @@ def __init__(
         when: Optional[str] = "before",
         anchor_phenotype: "Phenotype" = None,
     ):
-        if min_days is not None:
-            assert min_days.operator in [
-                ">",
-                ">=",
-            ], f"min_days operator must be > or >=, not {min_days.operator}"
-        if max_days is not None:
-            assert max_days.operator in [
-                "<",
-                "<=",
-            ], f"max_days operator must be > or >=, not {max_days.operator}"
-        if max_days is not None and min_days is not None:
-            assert (
-                min_days.value <= max_days.value
-            ), f"min_days must be less than or equal to max_days"
-        assert when in [
-            "before",
-            "after",
-        ], f"when must be 'before' or 'after', not {when}"
+        verify_relative_time_range_filter_input(min_days, max_days, when)
```

Comment:
hmm this is a hint to me that the abstraction isn't quite right. let's keep it for now but it's weird how much CCPT is overlapping with RTRF.